setdeviceconfig.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import yaml
+from shutil import move
+
+
+def split_yaml_params(yaml_raw, params):
+    """ 
+        returns yaml without specified params and yaml from params
+        keeping comments that are above parameters
+    """
+
+    comment_chunk = ""
+    clean_yaml = ""
+    removed_yaml = ""
+    param_regex = re.compile("^\\s*(\\S*):")
+    comment_regex = re.compile("^\\s*#")
+    for line in yaml_raw.splitlines(True):
+        if comment_regex.match(line):
+            comment_chunk += line
+        elif line.split() == "":
```

Comment:
`split` always returns a list so this will never be true

Reply:
whoops was mean't to be strip()